server/utils/licenceFactory.js

```diff
@@ -50,6 +51,23 @@ function createConditionsObject(selectedConditions, formInputs) {
     }, {});
 }
 
+function createEligibilityObject(object) {
+    const acceptedAttributes = Object.keys(licenceModel.eligibility);
+
+    const filterIfUnwanted = {
+        excludedReasons: 'excluded',
+        unsuitableReasons: 'unsuitable'
+    };
+
+    const filteredAcceptedAttributes = acceptedAttributes.filter(attribute => {
+        const decider = filterIfUnwanted[attribute];
```

Comment:
Suggest: const requiredSelectorChecked = requiredSelectionForAttribute[attribute]. Wordy but I find it tricky to follow currently